bin/npm-dependency-fallback.js

```diff
@@ -45,12 +63,34 @@ for (const dependencyName of Object.keys(packageJson.dependencies)) {
         }
     }
 
-    missingDependencies.push(dependency);
+    if (!isYarn) {
+        missingDependencies.push(dependency);
+    }
 }
 
 if (missingDependencies.length > 0) {
-    const dependenciesStr = missingDependencies.map(d => `${d.name}@${d.version}`).join(' ');
-    print(`installing ${dependenciesStr}...`);
-    childProcess.execSync(`npm install --no-save ${dependenciesStr}`);
+    if (isYarn) {
+        for (dependenciesStr of missingDependencies) {
+            const parsedToPath = dependenciesStr.toPath.split('/');
+            const fromPath = dependenciesStr.fromPath;
+            const toPath = dependenciesStr.toPath;
+            const currentDirectory = process.cwd();
+            const fullPath = `${currentDirectory}/${fromPath}`;
+
+            if (fs.existsSync(fullPath)) {
```

Comment:
How about turning this into a single `execSync()` ?
Doing 3 x N doesn't sounds great to me :)

Reply:
in fact for yarn I have to do the manipulation myself, it does not allow to install a package without adding it
so no